zerver/lib/actions.py

```diff
@@ -376,7 +376,7 @@ def stream_name(stream):
                           default_events_register_stream=default_events_register_stream_name,
                           default_all_public_streams=user_profile.default_all_public_streams,
                           avatar_url=avatar_url(user_profile),
-                          owner=user_profile.bot_owner.email,
+                          owner=user_profile.bot_owner.email if user_profile.bot_owner else None,
                           ))
```

Comment:
I added a blank line b/w commit message and commit description, but it didn't work for commit: 
`admin.js: Toggle visibility of 'open-user-form' on user/bot deactivation.`